PluginsAndFeatures/azure-toolkit-for-intellij/rider/ReSharper.Azure/src/Azure.Project/FunctionApp/FunctionAppProjectDetector.cs

```diff
@@ -110,7 +129,8 @@ public static bool IsAzureFunctionsProject([NotNull] IProject project)
                 .FirstNotNull());
 
             // 2) Check expected package reference.
-            var hasExpectedPackageReference = HasFunctionsPackageReference(project, targetFrameworkId);
+            var hasExpectedPackageReference = DefaultWorker.HasFunctionsPackageReference(project, targetFrameworkId) || 
```

Comment:
Could you please add some comment here with a reference to the function upgrade link to understand the difference between Isolated worker and default worker?